app/views/users/show.html.erb

```diff
@@ -1,10 +1,18 @@
-<p style="color: green"><%= notice %></p>
-
-<%= render @user %>
-
-<div>
-  <%= link_to "Edit this user", edit_user_path(@user) %> |
-  <%= link_to "Back to users", users_path %>
-
-  <%= button_to "Destroy this user", @user, method: :delete %>
-</div>
+<div class="posts">
+<img src="<%= @user.photo %>" />
+<h1><%= @user.name %></h1>
+<h2><%= @user.bio %></h2>
+<p><%= @user.posts_counter %></p>
+
+    <% @user.posts.each do |post| %>
+        <div class="schow_posts">
+        <h3><%=post.title %></h3>
+        <p><%=post.text %></p>
+        <p>comment: <%=post.comments_counter %>
+        <p>likes: <%=post.likes_counter %>
+        </div>
+    <% end %>

```

Comment:
- [ ] Following the project requirements, you should use the `#recent_post` method to render the user's 3 most recent posts.